pkg/xray/provider_test.go

```diff
@@ -82,6 +82,37 @@ func testAccDeleteRepo(t *testing.T, repo string) {
 	}
 }
 
+// Create a project. It will be used in the tests
+func testAccCreateProject(t *testing.T, projectKey string, projectName string) {
+	restyClient := getTestResty(t)
+
+	type Project struct {
+		DisplayName string `json:"display_name"`
+		Description string `json:"description"`
+		ProjectKey  string `json:"project_key"`
+	}
+
+	project := Project{}
+	project.DisplayName = projectName
+	project.Description = "Project created by TF provider test"
+	project.ProjectKey = projectKey
+	response, errProject := restyClient.R().SetBody(project).Post("/access/api/v1/projects")
+
+	if response.StatusCode() != http.StatusCreated {
+		t.Error(errProject)
+	}
+}
+
+// Delete test projects after testing
+func testAccDeleteProject(t *testing.T, projectKey string) {
+	restyClient := getTestResty(t)
+
+	response, errProject := restyClient.R().Delete("/access/api/v1/projects/" + projectKey)
+	if errProject != nil || response.StatusCode() != http.StatusNoContent {
```

Comment:
Not a blocker: I think `response.IsError()` is better than checking a specific status code.

Reply:
I think I'll leave it as is. I feel more comfortable knowing that the call should return 204.